tests/upgrades/test_repository.py

```diff
@@ -144,34 +137,29 @@ def test_pre_scenario_custom_repo_check(self, target_sat, save_test_data):
         lce = target_sat.api.LifecycleEnvironment(organization=org).create()
 
         product = target_sat.api.Product(organization=org).create()
-        target_sat.create_custom_html_repo(rpm1)
-        repo = target_sat.api.Repository(
-            product=product.id, url=f'{target_sat.url}/pub/custom_repo'
-        ).create()
+        repo = target_sat.api.Repository(product=product.id, url=settings.repos.yum_1.url).create()
         repo.sync()
-
-        content_view = publish_content_view(org=org, repolist=repo)
+        content_view = target_sat.publish_content_view(org, repo)
         content_view.version[0].promote(data={'environment_ids': lce.id})
-
-        subscription = target_sat.api.Subscription(organization=org).search(
-            query={'search': f'name={product.name}'}
-        )[0]
         ak = target_sat.api.ActivationKey(
             content_view=content_view, organization=org.id, environment=lce
         ).create()
-        ak.add_subscriptions(data={'subscription_id': subscription.id})
-        rhel7_client = Broker(container_host='ubi7:latest', host_class=ContentHost).checkout()
-        rhel7_client.install_katello_ca(target_sat)
-        rhel7_client.register_contenthost(org.label, ak.name)
-        rhid = rhel7_client.execute('subscription-manager identity')
+        if not target_sat.is_sca_mode_enabled(org.id):
+            subscription = target_sat.api.Subscription(organization=org).search(
+                query={'search': f'name={product.name}'}
+            )[0]
+            ak.add_subscriptions(data={'subscription_id': subscription.id})
```

Comment:
this check won't be required for 6.11.z and 6.12.z, non-blocking 
```suggestion
        subscription = target_sat.api.Subscription(organization=org).search(
             query={'search': f'name={product.name}'}
        )[0]
        ak.add_subscriptions(data={'subscription_id': subscription.id})
```

Reply:
it's better for consistency